lib/drivers/docs/tandemCGM.md

```diff
@@ -0,0 +1,97 @@
+## Checklist for CGM Implementation
+
+(Key:
+
+ - `[x]` available in data protocol/documented in spec and implemented
+ - `[-]` available in data protocol/documented in spec but *not* yet implemented
+ - `[?]` unknown whether available in data protocol/documented in spec; *not* yet implemented
+ - `*[ ]` TODO: needs implementation!
+ - `[ ]` unavailable in data protocol and/or not documented in spec and not yet implemented)
+
+### Required if Present
+
+#### CBG
+
+  - `[x]` cbg values
+  - `[ ]` units of cbg values (read from device, not hard-coded)
+  - `[x]` out-of-range values (LO or HI)
+  - `[x]` out-of-range value thresholds (e.g., often 40 for low and 400 for high on CGMs)
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+A status flag indicates if glucose reading was in response to calibration, which we store in payload with `{calibration_response : true}`.
+
+#### Device Events
+  - `[x]` calibrations
+    - `[x]` calibration value
+    - `[ ]` units of calibration value (read from device, not hard-coded)
+  - `[ ]` time changes (presence of which is also in the [BtUTC section](#bootstrapping-to-utc) below)
+    - `[ ]` device display time `from` (before change) and `to` (result of change)
+    - `[ ]` agent of change (`automatic` or `manual`)
+    - `[ ]` timezone
+    - `[ ]` reason for change (read from device)
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+Calibration events contain three additional timestamps (`timestamp`, `current_time` and `calibration_timestamp`) as well as a `calibration_reading` (which should match the manual SMBG value), which we store in payload.
+
+#### Settings
+
+  - `[ ]` units preference for BG display
+  - `[ ]` units of data being uploaded (will be mutated to mmol/L storage units if not mmol/L)
+  - `[-]` transmitter ID
+  - `[-]` low alert settings
+    - `[-]` enabled
+    - `[-]` level/threshold
+    - `[-]` snooze threshold
+  - `[-]` high alert settings
+    - `[-]` enabled
+    - `[-]` level/threshold
+    - `[-]` snooze threshold
+  - `[-]` rate-of-change alerts
+    - `[-]` fall rate alert
+        - `[-]` enabled
+        - `[-]` rate threshold for alerting
+    - `[-]` rise rate alert
+        - `[-]` enabled
+        - `[-]` rate threshold for alerting
+  - `[-]` out-of-range alerts
+    - `[-]` enabled
+    - `[-]` snooze time between alerts
+  - `[ ]` predictive alerts
+    - `[ ]` low prediction
+        - `[ ]` enabled
+        - `[ ]` time sensitivity (minutes to predicted low for alerting)
+    - `[ ]` high prediction
+        - `[ ]` enabled
+        - `[ ]` time sensitivity (minutes to predicted high for alerting)
+  - `[-]` calibration alerts/reminders
```

Comment:
Want to edit checklist draft since we discussed this and what you're referring to is not the setting (which doesn't exist) but the alert itself? As part of drafting CGM alert data models, I'll draft additions to the checklist.